internal/gitserver/gitdomain/common.go

```diff
@@ -230,14 +229,15 @@ func (m Message) Body() string {
 
 // A Hunk is a contiguous portion of a file associated with a commit.
 type Hunk struct {
-	StartLine uint32 // 1-indexed start line number
-	EndLine   uint32 // 1-indexed end line number
-	StartByte uint32 // 0-indexed start byte position (inclusive)
-	EndByte   uint32 // 0-indexed end byte position (exclusive)
-	CommitID  api.CommitID
-	Author    Signature
-	Message   string
-	Filename  string
+	StartLine        uint32 // 1-indexed start line number
+	EndLine          uint32 // 1-indexed end line number
+	StartByte        uint32 // 0-indexed start byte position (inclusive)
+	EndByte          uint32 // 0-indexed end byte position (exclusive)
+	CommitID         api.CommitID
+	PreviousCommitID api.CommitID
+	Author           Signature
+	Message          string
+	Filename         string
```

Comment:
Could we add a comment here what this is?

```suggestion
	// Filename is the name of the file at the commit prior to this change.
	Filename         string
```